src/controllers/features/sole-trader/whereDoYouLiveController.ts

```diff
@@ -26,14 +27,17 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
         // get data from mongo and save to session
         const acspData = await getAcspRegistration(session, session.getExtraData(SUBMISSION_ID)!, res.locals.userId);
         saveDataInSession(req, USER_DATA, acspData);
+
+        const { payload, countryInput } = new AmlSupervisoryBodyService().getCountryPayload(acspData);
         res.render(config.SOLE_TRADER_WHERE_DO_YOU_LIVE, {
```

Comment:
can you create new service to getCountryPayload instead of using AmlSupervisoryBodyService